packages/flutter_tools/lib/src/asset.dart

```diff
@@ -82,6 +83,7 @@ abstract class AssetBundle {
     String assetDirPath,
     @required String packagesPath,
     bool deferredComponentsEnabled = false,
+    TargetPlatform targetPlatform,
```

Comment:
@chingjun I unfortunately needed to add another parameter to AssetBundle.build. I took a quick look in cs and it doesn't seem like there's any AssetBundle subclasses that implements build.